src/pybind/mgr/cephadm/module.py

```diff
@@ -1523,19 +1523,19 @@ def remove_daemons(self, names):
         return self._remove_daemons(args)
 
     @trivial_completion
-    def remove_service(self, service_name):
+    def remove_service(self, service_name) -> str:
         self.log.info('Remove service %s' % service_name)
         self._trigger_preview_refresh(service_name=service_name)
         found = self.spec_store.rm(service_name)
         if found:
             self._kick_serve_loop()
-            return ['Removed service %s' % service_name]
+            return 'Removed service %s' % service_name
         else:
             # must be idempotent: still a success.
-            return [f'Failed to remove service. <{service_name}> was not found.']
+            return f'Failed to remove service. <{service_name}> was not found.'
```

Comment:
```suggestion
            return f'Failed to remove service. {service_name} was not found.'
```
minor nit..output looks weird with `<>` chars? remove these?